docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -166,25 +166,25 @@ The inline editing toggle lets you lock or unlock the canvas. When turned off th
 
 Canvases support [data links](ref:data-links) for all elements except drone and button elements. You can add a data link by following these steps:
 
-1. Set an element to be tied to a field value.
-1. Turn off the inline editing toggle.
-1. Create an override for **Fields with name** and select the element field name from the list.
-1. Click the **+ Add override property** button.
-1. Select `Datalinks > Datalinks` from the list.
-1. Click **+Add link** add a title and URL for the data link.
-1. Hover over the element to display the data link tooltip.
-1. Click on the element to be able to open the data link.
-
-If multiple elements use the same field name, and you want to control which elements display the data link, you can create a unique field name using the [add field from calculation transform](ref:add-field-from-calculation-transform). The alias you create in the transformation will appear as a field you can use with an element.
-
-1. In the panel editor for the canvas, click the **Transform** tab.
-1. Select **Add field from calculation** from the list of transformations, or click **+ Add transformation** to display the list first.
-1. Choose **Reduce row** from the dropdown and click the field name that you want to use for the element.
-1. Select **All Values** from the **Calculation** dropdown.
-1. Add an alias for the field name.
-1. Reference the new unique field alias to create the element and field override.
-
-{{< video-embed src="/media/docs/grafana/canvas-data-links-9-4-0.mp4" max-width="750px" caption="Data links demo" >}}
+1. Click on the element you want to add a data link to while inline editing is enabled.
+1. In either the inline editor or panel editor, expand the **Selected element** editor.
+1. In the selected element pane, scroll down to the **Data links** section and expand it.
```

Comment:
Are selected element pane and **Selected element** editor the same thing?

Reply:
Yes they are

Reply:
Okay, I think we can leave it out then since it's already referenced. 
```suggestion
1. Scroll down to the **Data links** section and expand it.
```